.gitignore

```diff
@@ -17,7 +17,7 @@ Carthage/Checkouts
 /documentation
 /build
 .build
-.swiftpm/
```

Comment:
Removed to fix [the Xcode 14 error](https://app.circleci.com/pipelines/github/mapbox/mapbox-navigation-ios/11548/workflows/374430ef-89ce-4dfb-ba35-d4359033832f/jobs/111957) when running `Package.swift` tests

> Details:  Error deleting scheme: Cannot modify data because the process disallows saving